tests/patches/detox+20.23.1.patch

```diff
@@ -0,0 +1,104 @@
+diff --git a/node_modules/detox/android/detox/src/full/java/com/wix/detox/reactnative/idlingresources/NetworkIdlingResource.java b/node_modules/detox/android/detox/src/full/java/com/wix/detox/reactnative/idlingresources/NetworkIdlingResource.java
```

Comment:
@Salakar - didn't see an PRs upstream related to this, but I have a reasonable working relationship with Detox, I could try to carry something in there if you could succinctly describe the thing you saw that you were working around here / working theory involved in the fix. Nice to be current on detox again and ideally we can mainline whatever patch hackery we're using